src/BibleWell.Aquifer.Data/Constants.cs

```diff
@@ -0,0 +1,17 @@
+using BibleWell.Storage;
+
+namespace BibleWell.Aquifer.Data;
+
+internal sealed class Constants(IStorageService _storageService)
+{
+    private const string DatabaseFilename = "BibleWellAquifer.db3";
+    private const string ApplicationDir = "BibleWell";
```

Comment:
It seems like the storage service should be responsible for this because other projects outside of `Data` may need to save information there.